src/mame/video/deco32.cpp

```diff
@@ -77,10 +77,10 @@ void deco32_state::allocate_rowscroll(int size1, int size2, int size3, int size4
 	m_pf_rowscroll[1] = make_unique_clear<uint16_t[]>(size2);
 	m_pf_rowscroll[2] = make_unique_clear<uint16_t[]>(size3);
 	m_pf_rowscroll[3] = make_unique_clear<uint16_t[]>(size4);
-	save_pointer(NAME(m_pf_rowscroll[0].get()), size1);
-	save_pointer(NAME(m_pf_rowscroll[1].get()), size2);
-	save_pointer(NAME(m_pf_rowscroll[2].get()), size3);
-	save_pointer(NAME(m_pf_rowscroll[3].get()), size4);
+	save_pointer(NAME(m_pf_rowscroll[0].get()), size1, 0);
+	save_pointer(NAME(m_pf_rowscroll[1].get()), size2, 1);
+	save_pointer(NAME(m_pf_rowscroll[2].get()), size3, 2);
+	save_pointer(NAME(m_pf_rowscroll[3].get()), size4, 3);
```

Comment:
Is the index necessary?  The `NAME` macro should already be giving distinct names.